presto-parquet/src/main/java/com/facebook/presto/parquet/ParquetTypeUtils.java

```diff
@@ -256,4 +243,20 @@ public static boolean isValueNull(boolean required, int definitionLevel, int max
     {
         return !required && (definitionLevel == maxDefinitionLevel - 1);
     }
+
+    public static long getShortDecimalValue(byte[] bytes)
```

Comment:
Is this copied from the hive module? If so, please add a comment that this is copied.

Reply:
yes, will do